.appveyor.yml

```diff
@@ -303,7 +303,7 @@ install:
         Invoke-WebRequest -Uri "https://win.rustup.rs" `
                           -OutFile "$env:TEMP\rustup-init.exe"
         Exec -NoNewLines {
-          & "$env:TEMP\rustup-init.exe" -y --default-toolchain 1.30.0
+          & "$env:TEMP\rustup-init.exe" -y --default-toolchain 1.31.0
```

Comment:
Did you need to clear the cache to get this to work? This wasn't upgrading when I had tried previously.

Reply:
@hayd Yes, appveyor.yml isn't smart enough to upgrade rust when a different version is wanted.
PS: if you want to try something without cache interference you can just comment out the relevant `CACHE` lines in your PR.